drake_cmake_installed/src/particles/test/utilities_test.cc

```diff
@@ -0,0 +1,108 @@
+#include "../utilities.h"
```

Comment:
I think I would prefer you add the parent directory to the list of include directories.